kw/platform/requests/__init__.py

```diff
@@ -0,0 +1,9 @@
+try:
+    from requests import *
```

Comment:
I'd avoid importing `*` if possible, I was thinking of this structure now:

#### `kw/platform/__init__.py`

```
from .proxies import patched_requests as requests
```

#### `kw/platform/proxies/__init__.py`

```
from .requests import patched_requests
```

#### `kw/platform/proxies/requests.py`

```
try:
    import requests
except ModuleNotFoundError:
     # something
```

…but this would be worse, cause we can't raise an import error when someone writes `from kw.platform import requests` :(

Do you see any alternatives maybe without `import *`? I don't.

Reply:
That's what I wanted to talk about. I don't see a way around the `import *`.

Also the separate source file `requests_utils.py` seems to me as the only solution to the option of manual patching of standard `requests` library like this
```python
from kw.platform.requests_utils import patch_with_user_agent
import requests

patch_with_user_agent('some-agent')
requests.get('http://kiwi.com')
```

Reply:
Yep, makes perfect sense, good job finding this design actually! 😊